pyfrc/physics/core.py

```diff
@@ -88,14 +86,18 @@ class PhysicsInterface:
     """
 
     @classmethod
-    def _create(cls, robot_path):
+    def _create_and_attach(
+        cls: type["PhysicsInterface"],
+        robot_class: type[wpilib.RobotBase],
```

Comment:
We'd need either `from __future__ import annotations` or use typing.Type instead.